app/assets/v2/js/cart.js

```diff
@@ -1402,59 +1406,32 @@ Vue.component('grants-cart', {
     });
 
     // Read array of grants in cart from localStorage
-    const grantData = CartData.loadCart();
-
-    // Make sure none have empty currencies, and if they do default to 0.001 ETH. This is done
-    // to prevent the cart from getting stuck loading if a currency is empty
-    grantData.forEach((grant, index) => {
-      if (!grant.grant_donation_currency) {
-        grantData[index].grant_donation_currency = 'ETH';
-        grantData[index].grant_donation_amount = '0.001';
-      }
-    });
-    CartData.setCart(grantData);
-    this.grantData = grantData;
-
-    // Initialize array of empty comments
-    this.comments = this.grantData.map(grant => undefined);
-
-    // Get list of all grant IDs and unique tokens in the cart
-    const grantIds = this.grantData.map(grant => grant.grant_id);
+    let grantData = CartData.loadCart();
+    const grantIds = grantData.map(grant => grant.grant_id);
 
-    // Fetch updated CLR curves for all grants
-    const url = `${window.location.origin}/grants/v1/api/grants?pks=${grantIds.join(',')}`;
+    // Fetch updated Grants data for all cart grants
+    const url = `${window.location.origin}/grants/v1/api/grants?pks=${grantIds.join(',')}&slim=true`;
     const response = await fetch(url);
-    const clrCurves = (await response.json()).grants;
+    const updatedGrant = (await response.json()).grants;
 
-    // Update CLR curves
-    this.grantData.forEach((grant, index) => {
-      // Find the clrCurves entry with the same grant ID as this grant
-      const clrIndex = clrCurves.findIndex(item => {
-        return Number(item.id) === Number(grant.grant_id);
-      });
+    grantData.forEach(async(grant, index) => {
 
-      // Update grantData from server
-      this.$set(this.grantData[index], 'grant_clr_prediction_curve', clrCurves[clrIndex].clr_prediction_curve);
-      this.$set(this.grantData[index], 'is_on_team', clrCurves[clrIndex].is_on_team);
+      const grantIndex = updatedGrant.findIndex(item => {
+        return Number(item.grant_id) === Number(grant.grant_id);
+      });
 
+      // Make sure none have empty currencies, and if they do default to 0.001 ETH. This is done
+      // to prevent the cart from getting stuck loading if a currency is empty
+      updatedGrant[grantIndex]['grant_donation_currency'] = grant.grant_donation_currency ? grant.grant_donation_currency : 'ETH';
+      updatedGrant[grantIndex]['grant_donation_amount'] = grant.grant_donation_amount ? grant.grant_donation_amount : '0.001';
```

Comment:
PR looks great! Just one small comment here: This should probably vary by tenant, right? It seems right now we set the default value to 0.001 ETH regardless of the grant's tenant

Reply:
only if not defined but I fixed the others areas to always be defined already